<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceConfigWidget</class>
 <widget class="QWidget" name="AppearanceConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="appearance_frm">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="element_lbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Element:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="element_cmb">
        <item>
         <property name="text">
          <string>Global: Font style</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global: Constraints descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global: Object selection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global: Position hint text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global: Position hint box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global: Objects type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global: Lock arc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global: Lock body</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Table: Schema name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Table: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Table: Columns box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Table: Extended attributes box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Table: Title box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rule: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rule: Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Index: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Index: Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trigger: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trigger: Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Constraint: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Constraint: Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Policy: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Policy: Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Schema name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: References box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Extended attributes box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Title box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Table / columns alias</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Referenced column</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Referenced table</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>View: Reference descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Textbox: Body</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column: Column name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column: Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column: Included / Inherited by relationship</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column: Protected</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (pk): Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (pk): Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (fk): Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (fk): Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (uq): Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (uq): Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (nn): Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Column (nn): Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relationship: Descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relationship: Label text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relationship: Label box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relationship: Attribute text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relationship: Attribute descriptor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tag: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tag: Body</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Placeholder: Body</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Foreign table: Schema name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Foreign table: Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Foreign table: Attributes box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Foreign table: Extended attributes box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Foreign table: Title box</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="font_lbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Font:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="colors_lbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Colors:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QFontComboBox" name="font_cmb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QDoubleSpinBox" name="font_size_spb">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>8.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="unity_lbl">
          <property name="text">
           <string> pt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="underline_chk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Underline</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="italic_chk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="bold_chk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>element_cmb</tabstop>
  <tabstop>font_cmb</tabstop>
  <tabstop>font_size_spb</tabstop>
  <tabstop>bold_chk</tabstop>
  <tabstop>italic_chk</tabstop>
  <tabstop>underline_chk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
